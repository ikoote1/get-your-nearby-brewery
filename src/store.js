import { configureStore } from '@reduxjs/toolkit';
import categoryReducer from './redux/category/categorySlice';

const store = configureStore(
  {
    reducer: {
      brewery: categoryReducer,
    },
  },
);

export default store;
