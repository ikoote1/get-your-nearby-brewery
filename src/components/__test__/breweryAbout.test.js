import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import React from 'react';
import BreweryAbout from '../breweryAbout';
import store from '../../store';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';

window.scrollTo = jest.fn();

describe('Details component', () => {
  it('should render the data correctly', async () => {
    window.scrollTo.mockClear();
    const component = render(
      <BrowserRouter>
        <Provider store={store}>
          <BreweryAbout />
        </Provider>
        ,
      </BrowserRouter>,
    );
    expect(component.asFragment()).toMatchSnapshot();
  });
});
