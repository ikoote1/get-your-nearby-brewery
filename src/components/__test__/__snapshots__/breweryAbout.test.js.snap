// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details component should render the data correctly 1`] = `
<DocumentFragment>
  <div>
    Loading
  </div>
  ,
</DocumentFragment>
`;
